views/json/page/default.php

```diff
@@ -11,10 +11,3 @@
 elgg_set_http_header("Content-Type: application/json;charset=utf-8");
 
 echo $vars['body'];
-
-// backward compatibility
-global $jsonexport;
```

Comment:
Please make sure core doesn't use this elsewhere. I think that's why I left this.